launcher/minecraft/PackProfile.cpp

```
@@ -737,6 +741,11 @@ void PackProfile::installCustomJar(QString selectedFile)
     installCustomJar_internal(selectedFile);
 }
 
+void PackProfile::installComponents(QStringList selectedFiles)
```

shouldn't it display a warning when the internal step fails (returns false)? maybe it could just forward the result out of this function so it gets handled in VersionPage.

---

None of the other functions do so I kept it as is.

---

doesn't mean they're doing it right :woman_shrugging:
silently failing is annoying to debug, especially if you don't know how the internals work